control/scamp.cc

```diff
@@ -91,12 +99,18 @@ SCAMP::SCAMP()
     if (mode.has_rudder_mode()) {
       rudder_mode_ = mode.rudder_mode();
     }
+    if (mode.has_ballast_mode()) {
+      ballast_mode_ = mode.ballast_mode();
+    }
     if (IsDisabled(RUDDER)) {
       raw_rudder_ = 90;
     }
     if (IsDisabled(WINCH)) {
       raw_winch_ = 90;
     }
+    if (IsDisabled(BALLAST)) {
```

Comment:
For the time being, can we act consistently if we get a mode for the ballast that isn't WiFi or disabled? Either by changing this to be `!IsManualWifi`or by changing line 103 above this to not update the ballast mode if it isn't one of the 2 valid modes? I don't want to accidentally send RC while the ballast is moving and have it keep moving without stopping.